vividus-plugin-applitools/src/main/java/org/vividus/visual/eyes/factory/ImageEyesFactory.java

```diff
@@ -39,18 +41,23 @@ public ImageEyesFactory(LogHandler logHandler)
 
     public Eyes createEyes(ApplitoolsVisualCheck applitoolsVisualCheck)
     {
-        Eyes eyes = new Eyes();
+        EyesRunner runner = new ImageRunner();
+        runner.setLogHandler(logHandler);
+
+        Eyes eyes = new Eyes(runner);
         eyes.setApiKey(applitoolsVisualCheck.getExecuteApiKey());
         String viewporSize = applitoolsVisualCheck.getViewportSize();
         //Environment
-        eyes.setExplicitViewportSize(viewporSize == null ? null : RectangleSize.parse(viewporSize));
+        if (viewporSize != null)
+        {
+            eyes.setViewportSize(RectangleSize.parse(viewporSize));
```

Comment:
I believe we set null for a reason, what eyes-5 sends to Applitools for viewport when viewportSize = null? ?

Reply:
there is null check now, if viewport size is not set then sdk takes current one

Reply:
how does it differ from v3?

Reply:
ops I messed it up, image Eyes is no longer a subclass of EyesBase, so by new logic viewport size is a required parameter